docs/src/content/docs/availablemodels.mdx

```diff
@@ -0,0 +1,35 @@
+---
+  title: "Available Models"
+---
+
+The Ultravox API currently provides the following models.
+
+<table class="w-full">
+    <tr class="w-full">
+        <th class="w-4/12">Model</th>
+        <th class="w-8/12">Description</th>
+    </tr>
+    <tr>
+        <td class="font-mono">fixie-ai/ultravox</td>
+        <td>This is an alias that points to `ultravox-70B`. The default if no model is specified when creating a call with the API.</td>
```

Comment:
Ideally we'd word this such that we don't have to update docs every time we change this.  Maybe something like "This is an alias for our most sophisticated model."

Reply:
yeah makes sense. gonna keep for now and revisit when we have more models and it's more complicated/harder to update.